okhttp/src/main/java/io/grpc/okhttp/CipherSuite.java

```diff
@@ -0,0 +1,379 @@
+/*
```

Comment:
Since we are public ConnectionSpec (I deprecated the API using com.squareup.okhttp.ConnextionSpec and added new API for our own ConnectionSpec), this has to be public.